crates/bevy_ecs/src/lib.rs

```diff
@@ -941,6 +941,27 @@ mod tests {
         );
     }
 
+    #[test]
+    fn try_insert_resource_no_collision() {
+        let mut world = World::default();
+        world.try_insert_resource(64u64);
+        assert_eq!(
+            *world.get_resource::<u64>().expect("Resource exists"),
```

Comment:
```suggestion
            *world.get_resource::<u64>().expect("Resource not found"),
```
The string you add in a `.expect()` is the error message you want to show, when you get `None` or `Err`

Reply:
Thanks, fixed this across all the expect calls in the tests